clusterloader2/testing/load/config.yaml

```diff
@@ -48,12 +48,34 @@
 {{$mediumDeploymentsPerNamespace := SubtractInt $mediumDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 {{$bigDeploymentsPerNamespace := SubtractInt $bigDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 
+# BEGIN scheduler-throughput section
+# Min number of pods per deployment to be used for measuring scheduler throughput
+# to get enough samples and accurate measurements in small clusters.
+{{$MIN_PODS_PER_DEPLOYMENT_TO_MEASURE_SCHEDULER_THROUGHPUT := 250}}
```

Comment:
Where is 250 coming from? Only the fact that it's 500 divisor? :D

What I'm mostly interested in is ensuring that we test large deployments (which is what we are doing in existing density test) - there are deployments of size 3000 there.
However, I think that constraining it to "cluster size" is actually reasonable - we will test 100 pod deployments in 100-node cluster and 5000-node deployments in 5k-node clusters.

So I personally think that instead of 250 here, I would simply use ".Nodes", just I would create many more namespaces there, to ensure that at least N(=500?) pods will be created.
[+ ensure that there are at least 2]